src/aasthaJJun20/ItemIDFromURL.java

```diff
@@ -0,0 +1,53 @@
+package aasthaJJun20;
+/*Assignment 33 : 22nd July 2020
+
+Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+public class ItemIDFromURL {
+	void findItemIDFromURL(String url) {
+		System.out.println("\nString input : " + url);
+		String temp = "";
+		try {
+			temp = url.substring(url.indexOf("itemId"));
```

Comment:
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}